ci-operator/config/openshift-assisted/assisted-events-scrape/openshift-assisted-assisted-events-scrape-master.yaml

```diff
@@ -33,13 +29,35 @@ tests:
 - as: lint
   commands: make lint
   container:
-    clone: true
-    from: assisted-events-scrape-build
+    from: src
 - as: unit-test
   commands: make unit-test
   container:
-    clone: true
-    from: assisted-events-scrape-build
+    from: src
+- as: integration-test
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 15m0s
+    version: "4.10"
+  steps:
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        sleep 3600
```

Comment:
is the `sleep` required or is it just to debug?

Reply:
Just debug, I will need more changes but I want to close https://github.com/openshift-assisted/assisted-events-scrape/pull/74 first, so I can better test it here